src/Workleap.OpenApi.MSBuild/msbuild/tools/Workleap.OpenApi.MSBuild.targets

```diff
@@ -17,10 +17,10 @@
   <!-- We can only analyze the OpenAPI spec after the project has been built -->
   <Target Name="ValidateOpenApi" DependsOnTargets="ResolveProjectReferences" AfterTargets="Build" Condition="'$(OpenApiEnabled)' == 'true'">
     <PropertyGroup>
-      <!-- Set development mode: ContractFirst or CodeFirst -->
-      <OpenApiDevelopmentMode Condition="'$(OpenApiDevelopmentMode)' == ''">ContractFirst</OpenApiDevelopmentMode>
+      <!-- Set development mode: ValidateContract or GenerateContract -->
+      <OpenApiDevelopmentMode Condition="'$(OpenApiDevelopmentMode)' == ''">GenerateContract</OpenApiDevelopmentMode>
 
-      <!-- When ContractFirst, validate if the provided specification files match the code -->
+      <!-- When ValidateContract, validate if the provided specification files match the code -->
```

Comment:
Now it also apply in GenerateContract: instead of generating the contract it will compare it to the code.

Reply:
SOme feedback where proposing to put a default value of true is we are compiling in release mode: that way on the CI it will compare by default.

Reply:
I'm not sure I understand what has to be done here but seems out of scope of this PR.

Reply:
We should at least remove this part "When ValidateContract" because the rest of the sentence is true for both mode.

Reply:
Fixed